xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoPPFFmpeg.cpp

```diff
@@ -96,7 +96,7 @@ void CDVDVideoPPFFmpeg::Process(VideoPicture* pPicture)
   }
 
   uint8_t* srcPlanes[YuvImage::MAX_PLANES], *dstPlanes[YuvImage::MAX_PLANES];
-  int srcStrides[YuvImage::MAX_PLANES];
+  int srcStrides[YuvImage::MAX_PLANES]{};
```

Comment:
Same thing here. Seems odd but I guess default init doesn't hurt.

Reply:
for the sake of some brackets to remove a warning from a static analysis tool, id just put it in personally.